pkg/yqlib/decoder_toml.go

```diff
@@ -286,7 +286,7 @@ func (dec *tomlDecoder) processTable(currentNode *toml.Node) (bool, error) {
 		}
 
 		runAgainstCurrentExp, err = dec.decodeKeyValuesIntoMap(tableNodeValue, tableValue)
-		if err != nil && !errors.Is(io.EOF, err) {
+		if err != nil && !errors.Is(err, io.EOF) {
```

Comment:
This one introduce a real bug fix and a change in the code behavior maybe.

Unless the error was not wrapped at all

Reply:
While that's more correct (now that I read the docs again) - afaik this doesn't fix a known bug as the err is an instance of io.EOF anyway